README.md

```diff
@@ -20,6 +20,40 @@ For development installation, we suggest following the github actions python-pac
 
 Currently, tests run on python 3.9, so those are the recommended versions if not installing via conda. BKNN doesn't currently install on Python 3.10 due to a skip in Bioconda.
 
+## How to update
+
+Updating the stack that depends on this script is unfortunately complex, it gets all the way to Galaxy tools using this for production pipelines in at least two different institutions.
+
+1. Open a branch here from develop
+2. Change the conda dependencies versions desired in the test-env.yaml file (if any)
+3. Make your changes in the branch (including adding any tests to the bats files for new areas or functionalities).
+4. Iterate with the CI on the branch until all test pass with the new changes and dependencies.
+5. Get reviews and merge to develop.
+6. Open a bump branch in bioconda, making sure that it points to the develop branch of this repo rather than the pypi release and that all dependencies in bioconda reflect what is currently in the test-env.yaml file of this repo (which was used to test in the feature branch here in points 2 to 4).
+7. Once the tests pass in bioconda, ask the bot to fetch the artifacts, download the linux artifact. Leave the bioconda branch in waiting, do not merge it (and probably add a message saying so)
```

Comment:
I am little bit confuse here for 

`Leave the bioconda branch in waiting, do not merge it`

Reply:
```suggestion
7. Once the tests passes in bioconda, ask the bot to fetch the artifacts, download the linux artifact (to run the  tests against that artifact or inside the artifact container). Leave the bioconda branch in draft and do not merge it (and probably add a message saying so) until you can verify that the created artifact can be used to run the tests (see steps below).
```

Reply:
Does that makes it clearer?

Reply:
So only merge when step 13 works fine?

Reply:
Can we merge this then @anilthanki ? Thanks.